File: backend/Models/IRS_Tax.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend.Models
{
  public class IRS_Tax
  {
    public int Id { get; set; }
    public double Employee_Social_Security { get; set; }
    public double Company_Social_Security { get; set; }
    public double Minimum_Wage { get; set; }
    public double A_La_Card { get; set; }
    public double Tta_Kms { get; set; }
    public int Car_Amortization { get; set; }
    public double Lunch { get; set; }
  }
}

File: web/src/app/_models/IRS_Tax.ts
export interface IRS_Tax {
  id: number;
  employee_Social_Security: number;
  company_Social_Security: number;
  minimum_Wage: number;
  a_La_Card: number;
  tta_Kms: number;
  car_Amortization: number;
  lunch: number;
  subsRefeicao?: number;
  minimumWage?: number;
  variable?: number;
  tta?: number;
  extraTax?: number;
  proposedDailyRate?: number;
  carType?: number;
  target_Margin?: number;
  numberOfMonths?: number;
  numberOfDaysMonth?: number;
}
